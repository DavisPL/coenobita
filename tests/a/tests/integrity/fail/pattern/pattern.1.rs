#![allow(unused_variables)]
#![allow(unused_assignments)]

fn main() {
    #[cnbt::integrity({*}{*} ({root}{root}, {b}{b}))]
    let (x, mut y) = (5, b::boolean());

    y = false;
}