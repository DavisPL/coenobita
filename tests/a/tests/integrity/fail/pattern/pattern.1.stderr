error: expected {b}{b}, found {root}{root}
 --> tests/integrity/fail/pattern/pattern.1.rs:8:9
  |
8 |     y = false;
  |         ^^^^^
